vim_onenote.ahk

```diff
@@ -196,41 +295,70 @@ Send, {ShiftUp}
 return
 
 ; Delete current line
+; dd handled specially, to delete newline.
 d::
-if IsLastKey("d")
-{
-    Send, {Home}{ShiftDown}{End}
-    Send, ^c ; Yank before delete, don't use cut so blank lines are deleted 
+    SelectMotion()
+    send ^x
+    if IsLastKey("d")
     Send, {Del}
-    Send, {Shift}
-}
-return 
+return
 
-+s::
++S::
 Send, {Home}{ShiftDown}{End}
 Send, ^x ; Cut instead of yank and delete
 Send, {ShiftUp}
 Gosub, InsertMode   
 return 
 
-return 
 ; TODO handle regular paste , vs paste something picked up with yy
```

Comment:
Not my comment :P I thought it was yours, at least you closed the issue about implementing that.
I think it is still relevant, I think the comment was relating to using an internal cut buffer (vim style), as opposed to the system buffer.
Personally I'm happy to leave it, at least until I get around to implementing different registers.